package.json

```diff
@@ -114,7 +114,9 @@
       "postbump": "gulp replaceVersion && git add ."
     }
   },
-  "dependencies": {},
+  "dependencies": {
+    "js-cookie": "^2.2.1"
```

Comment:
plz add the js-cookie dependency as devDependency

Reply:
Done